### DISCUSSION TITLE
Project migration

### DISCUSSION THREAD

**Discussion by billz** [RaspAP/raspap-webgui#801] [General]
You may have noticed a new GitHub home for this project. RaspAP has been migrated from my personal account ([billz](https://github.com/billz)) to a GitHub organization, where it belongs! From now on you'll find everything at https://github.com/RaspAP.

### Why the change?
A (very) brief history: This project traces its origins to a [blog post](http://sirlagz.net/2013/02/06/script-web-configuration-page-for-raspberry-pi/) written by @SirLagz nearly eight years ago to the day. He [followed up](http://sirlagz.net/2013/02/08/raspap-webgui/) by releasing the original `raspap-webgui` source code, saying "I have yet to decide on a good name for it". As often happens, the name stuck and it persists to this day.

My initial contribution was simply to wrap his code with Bootstrap and post it to GitHub. Since then, many [contributors](https://github.com/RaspAP/raspap-webgui/graphs/contributors) have been drawn to it, features added, support extended to more OS's, real [documentation](https://docs.raspap.com/) written, and several new repos created as the project has grown in popularity and scope.

On one level, it made sense to aggregate all this under the aegis of something other than an individual (ie., me). GitHub makes it ridiculously easy to transfer ownership of repos, maintaining redirects and so on. GitHub organizations are also great at centralizing code, facilitating collaboration and discoverability, more than I could ever do with a collection of personal repos.

More than this, though, it better reflects the contributions made by **_you_**—the community that has evolved around this project. 

I've committed [changes](https://github.com/RaspAP/raspap-webgui/commit/8a4453b0f5ab98cab64607c65eb87c2c7aea5560) to handoff existing repos to the new org, so the installer and everything else should work as before. If you encounter any hiccups post a comment below. Thanks.

**Comment by jrmhaig** [comment]
Hi Bill.

Great to see how this project has developed in the last few years. Thanks for the invitation to join the organisation. However, it has been some years since I made any contributions so it doesn't make sense for me to join but I don't want you to think that is because I have ignored it.

I'm glad to see that you are still using 10.3.141.* as the default network address range. That's probably the only thing I added that still survives. :-)

Good luck with the project.

**Comment by billz** [comment]
@jrmhaig I think you'd be surprised just how much your contributions have shaped the project. Indeed, you created the 'default config' (which now has a [dedicated page](https://docs.raspap.com/defaults/) in the docs), added authentication, CSRF tokens and loads of other key pieces. Your commits gave the project momentum in the early days that helped it attract a following.
